vm/http_test.go

```diff
@@ -37,7 +37,19 @@ func TestHTTPRequest(t *testing.T) {
 
 		res = Net::HTTP.head("http://127.0.0.1:3000/index")
 		res["Content-Length"]
-		`, "15"},
+		`, "16"},
```

Comment:
can we use the exact content instead of the length of the content for comparison?

Reply:
Currently head / delete / options only returns HTTP headers. I have post a thought on slack.

Reply:
ah of course, my fault 🤦 
can we just test the status code then (should all be `204`)

Reply:
Sorry late for the reply, just changed it to check the method from response headers.